server/TracySourceView.cpp

```diff
@@ -3843,69 +3843,48 @@ void SourceView::RenderAsmLine( AsmLine& line, const AddrStat& ipcnt, const Addr
     memcpy( buf, line.mnemonic.c_str(), msz );
     memset( buf+msz, ' ', m_maxMnemonicLen-msz );
     bool hasJump = false;
+    int jumpLabel = 0;
     if( line.jumpAddr != 0 )
     {
         auto lit = m_locMap.find( line.jumpAddr );
         if( lit != m_locMap.end() )
         {
+            jumpLabel = lit->second;
             char tmp[64];
-            sprintf( tmp, ".L%" PRIu32, lit->second );
-            strcpy( buf+m_maxMnemonicLen, tmp );
+            sprintf( tmp, ".L%" PRIu32, jumpLabel );
+            strcpy(buf+m_maxMnemonicLen, tmp);
             hasJump = true;
         }
     }
     if( !hasJump )
     {
-        memcpy( buf+m_maxMnemonicLen, line.operands.c_str(), line.operands.size() + 1 );
+        memcpy( buf+m_maxMnemonicLen, line.operands.string.c_str(), line.operands.string.size() + 1 );
     }
 
     const bool isInContext = IsInContext( worker, line.addr );
-    if( asmIdx == m_asmSelected )
-    {
-        TextColoredUnformatted( ImVec4( 1, 0.25f, 0.25f, isInContext ? 1.f : 0.5f ), buf );
-    }
-    else if( line.regData[0] != 0 )
-    {
-        bool hasDepencency = false;
-        int idx = 0;
-        for(;;)
-        {
-            if( line.regData[idx] == 0 ) break;
```

Comment:
Also worth noting that I removed this code path ; Unsure how to visualize this if it is desirable to bring back

Reply:
Why was it removed?

Reply:
Was unsure how to make it immediately obvious _what_ was being represented. 

Dimming things out of focus is a familiar concept to most, but unsure how to represent this - I also was not sure exactly what this was representing / what the idea behind it was.

Reply:
And what would this "this" be?

Reply:
Wasn't sure how to make the visualisation make sense. I've added it back locally to just be not dimmed. I.e. the logic is just
```c++
    const bool isInContext = IsInContext( worker, line.addr );
    const auto selected = m_asmSelected == asmIdx;
    auto transitiveDependency = false;
    for(int i = 0; line.regData[i] != 0; i++) {
        transitiveDependency |= line.regData[i] & ( WriteBit | ReadBit );
    }

    const auto shouldFocus = (isInContext && m_asmSelected == -1) || selected || transitiveDependency;
    const auto& colorPalette = shouldFocus ? AsmSyntaxColors : AsmSyntaxColorsDimmed;
```